common/app.py

```diff
@@ -13,20 +14,20 @@
 
 
 class App:
-    def __init__(self, name, path=None, deployments={}, secrets={}, load_secrets=True, account_id=''):
```

Comment:
Mutable references in arguments is something we need to avoid so I refactored this.